scripts/generators/stateless_validation_helper_generator.py

```diff
@@ -1001,24 +1003,27 @@ def genStructBody(self, struct: Struct, non_prop_feature: bool):
                 '''
 
         elif struct.sType in self.stype_version_dict.keys():
-            ext_name = self.stype_version_dict[struct.sType]
+            ext_names = self.stype_version_dict[struct.sType]
 
             # Skip extensions that are not in the target API
             # This check is needed because parts of the base generator code bypass the
             # dependency resolution logic in the registry tooling and thus the generator
             # may attempt to generate code for extensions which are not supported in the
             # target API variant, thus this check needs to happen even if any specific
             # target API variant may not specifically need it
-            if ext_name not in self.vk.extensions:
+            ext_names.intersection(self.vk.extensions.keys())
+            if len(ext_names) == 0:
                 return ""
 
             # Dependent on enabled extension
-            extension = self.vk.extensions[ext_name]
-            extension_check = ''
-            if extension.device:
-                extension_check = f'if ((is_physdev_api && !SupportedByPdev(physical_device, vvl::Extension::_{extension.name})) || (!is_physdev_api && !IsExtEnabled(device_extensions.{extension.name.lower()}))) {{'
-            else:
-                extension_check = f'if (!IsExtEnabled(instance_extensions.{extension.name.lower()})) {{'
+            extension_conditinals = list()
+            for ext_name in ext_names:
+                extension = self.vk.extensions[ext_name]
+                if extension.device:
+                    extension_conditinals.append( f'((is_physdev_api && !SupportedByPdev(physical_device, vvl::Extension::_{extension.name})) || (!is_physdev_api && !IsExtEnabled(device_extensions.{extension.name.lower()})))' )
+                else:
+                    extension_conditinals.append( f'!IsExtEnabled(instance_extensions.{extension.name.lower()})' )
+            extension_check = f'if ({' && '.join(extension_conditinals)}) {{'
```

Comment:
```suggestion
            extension_check = f'if ({" && ".join(extension_conditinals)}) {{'
```

should work... this is a python 3.12 feature I think but we are still at 3.10 python as the minimum requirements

Reply:
(CI is at 3.10 which is failing on this)

Reply:
I squashed it in